src/components/TreeWalker/constants.ts

```diff
@@ -1,5 +1,7 @@
 export const ACTIVE = 'Active';
+export const FEATURE = 'Feature';
 export const FIND_BY_ID = 'location/findByJurisdictionIds';
 export const FIND_BY_PROPERTIES = 'location/findByProperties';
 export const LOCATION = 'location';
+export const COULDNT_LOAD_CHILDREN = 'Could not load children';
 export const COULDNT_LOAD_PARENTS = 'Could not load parents';
```

Comment:
Should this strings be here or in the lang file?

Reply:
This component will be moved to OpenSRP web so it has its own constants module.  In this commit, we get the strings for display from the lang module: https://github.com/onaio/reveal-frontend/pull/1003/commits/7ee298e7766bb3da593ebe985be644bae1b47fa8